types/dinero.js/index.d.ts

```diff
@@ -29,13 +34,13 @@ declare namespace DineroFactory {
         getLocale(): string;
         setLocale(newLocale: string): Dinero;
         getPrecision(): number;
-        convertPrecision(newPrecision: number): Dinero;
+        convertPrecision(newPrecision: number, roundingMode?: RoundingMode): Dinero;
         add(addend: Dinero): Dinero;
         subtract(subtrahend: Dinero): Dinero;
         multiply(multiplier: number, roundingMode?: RoundingMode): Dinero;
         divide(divisor: number, roundingMode?: RoundingMode): Dinero;
         percentage(percentage: number): Dinero;
-        allocate(ratios: number[]): Dinero[];
+        allocate(ratios: readonly number[]): Dinero[];
```

Comment:
If you use `ReadonlyArray<number>` instead of `readonly number[]`, this will remain compatible with earlier versions of TypeScript.